Tests/Blazorise.Tests/Helpers/BlazoriseConfig.cs

```diff
@@ -30,7 +30,8 @@ public static void AddBootstrapProviders( TestServiceProvider services )
         services.AddSingleton<ValidatorValidationHandler>();
         services.AddSingleton<PatternValidationHandler>();
         services.AddSingleton<DataAnnotationValidationHandler>();
-        services.AddSingleton<IDateTimeFormatConverter, DateTimeFormatConverter>();
+        services.AddSingleton<IDateTimeDisplayFormatConverter, DateTimeDisplayFormatConverter>();
+        services.AddSingleton<IDateTimeInputFormatConverter, DateTimeInputFormatConverter>();
```

Comment:
Do we really need two versions? Are there different formats for Display/Input? Seems like it would always be the same format usage regardless?

Reply:
We have them separated because we use different libraries and they both have different format tokens. IDateTimeDisplayFormatConverter is for flatpickr. IDateTimeInputFormatConverter is for inputmask.

Reply:
haha that's even more confusing naming wise. I'd rename to something more udnerstandle in my opnion

Reply:
Do you have an example?

Reply:
I have renamed interfaces and implementations. Also moved them to `Vendors` namespace so that they are not visible at the `Blazorise` namespace level.